ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/Builders/DotNetBuildTaskBuilder.cs

```diff
@@ -0,0 +1,41 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks.Builders
+{
+    public class DotNetBuildTaskBuilder
+    {
+        private DotNetBuildTask dotnetBuildTask;
+
+        public DotNetBuildTaskBuilder()
+        {
+            dotnetBuildTask = new DotNetBuildTask
+            {
+                Run = "dotnet build"
+            };
+        }
+
+        public DotNetBuildTaskBuilder WithName(string name)
+        {
+            dotnetBuildTask.Name = name;
+            return this;
```

Comment:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/4.%20Methods.md#111-returns

Do the same for the rest please

Reply:
Thank you. Visual Studio reformatted my files automatically after I moved them to a new folder. :(